css/home.css

```diff
@@ -1,49 +1,171 @@
 /* Home page specific styles */
+@media only screen and (min-width: 360px) and (max-width: 960px) {
+  .footer {
+    padding: 10px;
+    padding-bottom: 4px;
+    background-color: #003c8f;
+    width: 215px;
+    heigth: 175px;
+    margin: 0 auto;
+    overflow: hidden;
+    position: relative;
+  }
 
-.homepage-hero {
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  width: 100%;
-  height: 100vh;
-  font-family: "roboto", sans-serif;
-  background: url("../assets/images/frente1.JPG") no-repeat center;
-  background-size: cover;
-}
+  .footer__text {
+    word-break: break-word;
+    text-align: center;
+    font-size: 16px;
+    color:white;
+    font-family: Roboto;
+  }
 
-.homepage-hero__layer {
-  height: 100%;
-  background-color: rgba(0, 0, 0, 0.6);
-}
+  .footer__social-media:nth-last-child(1) {
+    margin: 0;
+    display: inline-block;
+    padding:34px;
+  }
 
-.homepage-hero__article {
-  margin-right: 5%;
-  margin-left: 5%;
-  position: absolute;
-  bottom: 10%;
-}
+  .footer__footnote{
+    color:white;
+  }
 
-.homepage-hero__headline {
-  margin-bottom: 5px;
-  color: #ffffff;
-  font-size: 36px;
-  font-weight: bold;
-}
+  .footer__icon{
+    margin-right: 17px;
+  }
 
-.homepage-hero__headline h1 {
-  margin: 0;
-}
+  .homepage-hero {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    font-family: "roboto", sans-serif;
+    background: url("../assets/images/frente1.JPG") no-repeat center;
+    background-size: cover;
+  }
+
+  .homepage-hero__layer {
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.6);
+  }
 
-.homepage-hero__divider {
-  width: 90px;
-  height: 1px;
-  margin-bottom: 5px;
-  background-color: #73e8ff;
+  .homepage-hero__article {
+    margin-right: 5%;
+    margin-left: 5%;
+    position: absolute;
+    bottom: 10%;
+    width: 100%;
+  }
+
+  .homepage-hero__headline {
+    margin-bottom: 5px;
+    color: #ffffff;
+    font-size: 36px;
+    font-weight: bold;
+  }
+
+  .homepage-hero__headline h1 {
+    margin: 0;
+  }
+
+  .homepage-hero__divider {
+    width: 90px;
+    height: 1px;
+    margin-bottom: 5px;
+    background-color: #73e8ff;
+  }
+
+  .homepage-hero__promotext {
+    color: #ffffff;
+    font-size: 12px;
+    font-weight: regular;
+  }
 }
+@media only screen and (min-width: 960px) {
+ .footer {
+   padding: 10px;
+   padding-bottom: 4px;
+   background-color: #003c8f;
+   width: 1366px
+   heigth: 270px
+   margin: 0 auto;
+   overflow: hidden;
+   position: relative;
+ }
+
+ .footer__text {
+   word-break: break-word;
+   text-align: center;
+   font-size: 16px;
+   color:white;
+   font-family: Roboto;
+ }
 
-.homepage-hero__promotext {
-  color: #ffffff;
-  font-size: 12px;
-  font-weight: regular;
-}
\ No newline at end of file
+ .footer__social-media:nth-last-child(1) {
```

Comment:
what are you doing on this line?

Reply:
is style exclusive for the last item in the footer__social-media; this will be update in my next commit.